app/views/recipes/show.html.erb

```diff
@@ -1,17 +1,92 @@
-<p style="color: green"><%= notice %></p>
+<div class="card">
+  <div class="card-image">
+    <figure class="image is-3by1">
+      <%= image_tag @recipe.picture if @recipe.picture.attached? %>
+    </figure>
+  </div>
+  <div class="card-content">
+    <div class="media">
+      <div class="media-content">
+        <div class="level">
+          <div class="level-left is-8">
+            <div class="level-item">
+              <h2 class="title has-text-primary">
+                <%= @recipe.name %>
+              </h2>
+            </div>
+            <div class="level-item">
+              <p class="subtitle has-text-grey-dark">
+                <%= @recipe.recipe_category.name %>
+              </p>
+            </div>
+          </div>
+          <div class="level-right is-4 has-text-right">
+            <div class="level-item">
+              <h3 class="has-text-primary">
+                by: <%= @recipe.user.name %>
+              </h3>
+            </div>
+          </div>
+        </div>
+        <div class="level">
+          <div class="level-left is-8">
+            <div class="level-item">
+              <p class="subtitle is-6 has-text-dark">
+                <%= @recipe.description %>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
 
-<%= render @recipe %>
-
-<div>
-
-  <% if policy(@recipe).update? %>
-    <%= link_to "Edit this recipe", edit_recipe_path(@recipe) %>
-  <% end %>
-  <%= link_to "Back to recipes", recipes_path %>
-
-  <% if policy(@recipe).destroy? %>
-    <%= button_to "Destroy this recipe", @recipe, method: :delete %>
-  <% end %>
-
-  
+    <div class="content">
+      <div class="columns">
+        <div class="column is-8">
+          <h4 class="title has-text-primary">Ingredients</h4>
+          <ul>
+            <% @ingredients_json.each do |ingredient| %>
+              <%= tag.li "#{ingredient["name"].titleize}: #{ingredient["quantity"]} #{ingredient["quantity"].to_i > 1 ? ingredient["measure"].pluralize : ingredient["measure"]}" %>
+            <% end %>
+          </ul>
+        </div>
+        <div class="column is-4">
+          <h4 class="title has-text-primary">Appliances</h4>
+          <ul>
+            <% @recipe.appliances.each do |appliance| %>
```

Comment:
Did you remember to include the other table to prevent n+1 complexity? 👀

Reply:
Since appliances is not a foreign key but an array of appliance names, there's no other table to reference and no need to include an inclusive include here. xD